.github/workflows/build-on-release.yaml

```diff
@@ -152,7 +155,7 @@ jobs:
           TAG_NAME=${{ github.ref_name }}
           awk 'NR==6{$0="        url \"https://github.com/robusta-dev/holmesgpt/releases/download/'"$TAG_NAME"'/holmes-macos-latest-'"$TAG_NAME"'.zip\""}1' ./Formula/holmesgpt.rb > temp && mv temp ./Formula/holmesgpt.rb
           awk 'NR==7{$0="        sha256 \"'$MAC_BUILD_HASH'\""}1' ./Formula/holmesgpt.rb > temp && mv temp ./Formula/holmesgpt.rb
-          awk 'NR==9{$0="        url \"https://github.com/robusta-dev/holmesgpt/releases/download/'"$TAG_NAME"'/holmes-ubuntu-latest-'"$TAG_NAME"'.zip\""}1' ./Formula/holmesgpt.rb > temp && mv temp ./Formula/holmesgpt.rb
+          awk 'NR==9{$0="        url \"https://github.com/robusta-dev/holmesgpt/releases/download/'"$TAG_NAME"'/holmes-ubuntu-20.04-'"$TAG_NAME"'.zip\""}1' ./Formula/holmesgpt.rb > temp && mv temp ./Formula/holmesgpt.rb
           awk 'NR==10{$0="        sha256 \"'$LINUX_BUILD_HASH'\""}1' ./Formula/holmesgpt.rb > temp && mv temp ./Formula/holmesgpt.rb
```

Comment:
I dont think this is related but it looks like there are issues with calculating the hash for ubuntu, it looks like its from before this too, might want to verify brew install works on ubuntu
<img width="1090" alt="Screen Shot 2024-06-10 at 16 43 06" src="https://github.com/robusta-dev/holmesgpt/assets/97387909/42221265-e844-4163-92d0-e1ed0e2dcf32">

Reply:
Very nice catch. Fixed it.